### ISSUE TITLE
Multiple elements are only partially announced by iOS/VoiceOver

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#138671]
## Background/Context

VoiceOver on iOS is known to break up announcements by each tag or line break, which can result in requiring multiple swipes to hear an entire announcement or announcements missing some text (particularly when navigating by link or heading). 

It's possible to adjust this behavior using aria roles, such as `role="text"`. However, these roles can override other accessibility semantics and cause higher severity accessibility issues and should be thoroughly tested.

Ideally, any fix for this issue will be implemented consistently across all known occurrences (links to the VoiceOver behavior of each instance are included below):
- Representation Requests:
  - [Location on card and details page (ex: "Moca, PR 00676")](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126271%20-%20Representation%20Requests/Screenshots%20%26%20Recordings/126271%20-%2021%20-%20multiple%20swipes%20required.MP4?csf=1&web=1&e=VkomI7&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
  - [Heading level 1 on the details page](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126271%20-%20Representation%20Requests/Screenshots%20%26%20Recordings/126271%20-%2022%20-%20heading%20partially%20announced.MP4?csf=1&web=1&e=209OYD&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
- Find Claimant:
  - [Results summary](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126270%20-%20Find%20Claimant/Screenshots%20%26%20Recordings/126270%20-%2016%20-%20results%20summary%20requires%20multiple%20swipes.MP4?csf=1&web=1&e=bnvVCe&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
  - [Card headings](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126270%20-%20Find%20Claimant/Screenshots%20%26%20Recordings/126270%20-%2017%20-%20incomplete%20heading%20announcement.MP4?csf=1&web=1&e=uQtER6&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
- Homepage:
  - ["Watch a video of how the portal works (opens in a new tab)" link](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/125813%20-%20Home,%20Help,%20Sign%20In/Screen%20Reader%20Recordings/125813%20-%2026%20-%20YouTube%20link%20announced%20separately.MP4?csf=1&web=1&e=f4sH4z&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
- 526ez and 686c form flows
  - ["Download VA Form 21-526EZ (PDF)" and "Download VA Form 21-686c (PDF)" links](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126268%20-%20Submissions%20%26%20526EZ/Recordings%20%26%20Screenshots/126268%20-%2082%20-%20announced%20as%20multiple%20links.MP4?csf=1&web=1&e=Ea5rvL&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)
  - ["Upload VA Form 21-526EZ" and "Upload VA Form 21-686c" headings on the review page](https://dvagov.sharepoint.com/:v:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126268%20-%20Submissions%20%26%20526EZ/Recordings%20%26%20Screenshots/126268%20-%2090%20-%20heading%20partially%20announced.MP4?csf=1&web=1&e=HdMTG5&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)

If a consistent fix isn't possible, spot fixes would be acceptable as long as they don't cause any other accessibility regressions.
